### ISSUE TITLE
prompts.csv file parsing error

### ISSUE THREAD

**Issue by lencx** [f/awesome-chatgpt-prompts#130]
csv files containing spaces after commas cause panic when parsing (e.g. [rust-csv](https://github.com/BurntSushi/rust-csv)), is it possible to standardize the format. Similar problems may occur in other languages when parsing `prompts.csv`. Just a suggestion, writing your own program is also possible to handle 😅. 

The problematic line numbers are currently `120` and `124`.

```bash
thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: Error(UnequalLengths { pos: Some(Position { byte: 55222, line: 120, record: 119 }), expected_len: 2, len: 5 })'
```

<img width="1440" alt="Screenshot 2022-12-18 at 17 06 17" src="https://user-images.githubusercontent.com/16164244/208289891-31f50e81-93ad-4168-888d-0e20d69af159.png">

**Comment by f** [comment]
I am on it

**Comment by f** [comment]
@lencx can you check again?

**Comment by lencx** [comment]
Thanks, it's working now. I have parsed the data and will be able to release a new version soon.
<img width="600" alt="cmd" src="https://user-images.githubusercontent.com/16164244/208311459-1c2b1138-e7d9-452a-9ef7-15efaaf6e467.png">

**Comment by f** [comment]
Please let me know and I'll put it into README.

**Comment by lencx** [comment]
I have released `v0.5.0`, step:
`Menu -> Preferences -> Control Center -> Sync Prompts -> click Sync button -> Restart ChatGPT`
